README.md

```diff
@@ -1,25 +1,85 @@
 # Drainpipe
 
-## Dependencies
-This project requires [Task](https://github.com/go-task/task), [version 3 or later](https://taskfile.dev/#/taskfile_versions) to be installed.
+Drainpipe is a composer package which provides a number of build tool helpers
+for a Drupal site, including:
 
-## Setup
-An initial `Taskfile.yml` will be created when installing this package. Be sure to commit it to version control. To simplify future updates, consider including additional Taskfiles instead of adding commands to the file directly.
+- Site and database updates
+- Artefact packaging for deployment to a hosting provider
+- Test setups
 
-## Usage
-Below are some examples based on task files included in this package for certain project types.
+## Installation
 
-### Drupal
 ```
-task drupal:update
+composer require lullabot/drainpipe
+composer require lullabot/drainpipe-dev --dev
```

Comment:
```suggestion
# Includes development dependencies, but only in the `require-dev` section. This step is required for Drainpipe to function correctly.
composer require lullabot/drainpipe-dev --dev
```

Unless it's actually optional, in which case we should call it out!

Reply:
It's not required for it to functional correctly!